### PR TITLE
Merge v4.3.6 into master

### PR DESCRIPTION
**Pull request by teddytpc1**
This PR merges `v.4.3.6` into `master`.

### CODE REVIEW

**Review by vcerenu — APPROVED** [review]
LGTM

File: roles/wazuh/wazuh-dashboard/vars/debian.yml [hunk 1]
```diff
@@ -1,3 +1,7 @@
 ---
 
+<<<<<<< HEAD
+dashboard_version: 4.3.6
+=======
 dashboard_version: 4.5.0
+>>>>>>> master
```
Comment (okynos) [inline roles/wazuh/wazuh-dashboard/vars/debian.yml:7]: Fix this.

**Review by okynos — CHANGES_REQUESTED** [review]
Minor fix

File: roles/wazuh/wazuh-dashboard/vars/debian.yml (hunk 1 above)
Comment (teddytpc1) [reply roles/wazuh/wazuh-dashboard/vars/debian.yml]: Fixed. Thanks.

**Review by okynos — APPROVED** [review]

**Pull request merged**